js/upload-helper.js

```diff
@@ -69,6 +69,72 @@ var FileList = {
 		});
 	},
 
+
+	/**
+	 * Create an empty file inside the current album.
```

Comment:
"Create a `.nomedia`  file" instead maybe? And explaining that this hides the current album.

I've looked at Filelist and there is another method to create albums:
https://github.com/owncloud/core/blob/a7e7f5e18098ff0f536e58a7e1e5dfdc825e9086/apps/files/js/filelist.js#L2011